src/inheritAttributes.js

```diff
@@ -215,6 +216,68 @@ export const parseCaptionServiceMetadata = (service) => {
 
       return {channel, language};
     });
+  } else if (service.schemeIdUri === 'urn:scte:dash:cc:cea-708:2015') {
+    const values = service.value.split(';');
+
+    return values.map((value) => {
+      // service or channel number 1-63
+      let channel;
```

Comment:
Should these all reside within a `flags` object?

Reply:
yeah, might make it a bit easier, then we can just return the flags object below